cmd/anti_affinity_group_list.go

```diff
@@ -45,22 +43,19 @@ func (c *antiAffinityGroupListCmd) cmdPreRun(cmd *cobra.Command, args []string)
 }
 
 func (c *antiAffinityGroupListCmd) cmdRun(_ *cobra.Command, _ []string) error {
-	ctx := exoapi.WithEndpoint(
-		gContext,
-		exoapi.NewReqEndpoint(account.CurrentAccount.Environment, account.CurrentAccount.DefaultZone),
-	)
+	ctx := gContext
 
-	antiAffinityGroups, err := globalstate.EgoscaleClient.ListAntiAffinityGroups(ctx, account.CurrentAccount.DefaultZone)
+	antiAffinityGroups, err := globalstate.EgoscaleV3Client.ListAntiAffinityGroups(ctx)
 	if err != nil {
 		return err
 	}
 
 	out := make(antiAffinityGroupListOutput, 0)
 
-	for _, t := range antiAffinityGroups {
+	for _, t := range antiAffinityGroups.AntiAffinityGroups {
 		out = append(out, antiAffinityGroupListItemOutput{
-			ID:   *t.ID,
-			Name: *t.Name,
+			ID:   t.ID.String(),
+			Name: t.Name,
```

Comment:
same  here for ID type, if not possible use `.String()` from ID

Reply:
Here it was possible to remove the `.String()`